### PR TITLE
AG-165: Copy dev changes to staging

### PR DESCRIPTION
**Pull request by emmanmills**
1. Update EC2InstanceType to 't3.large' for staging
2. @zaro0508, please confirm if the changes in `config/misc/agora-bastian-docdb.yaml` and `templates/bastian2.yaml` are okay to go to staging. Thank you.

### CODE REVIEW

File: config/misc/agora-bastian-docdb.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Provision an EC2 bastian host for agora
+template_path: bastian2.yaml
+stack_name: agora-bastian-docdb
```
Comment (zaro0508) [inline config/misc/agora-bastian-docdb.yaml:3]: there's already [one bastian](https://github.com/Sage-Bionetworks/Agora-infra/blob/develop/config/misc/agora-bastian.yaml).  is there a reason for wanting to create another bastian?

File: templates/bastian2.yaml [hunk 2]
```diff
@@ -0,0 +1,140 @@
+AWSTemplateFormatVersion: 2010-09-09
```
Comment (zaro0508) [inline templates/bastian2.yaml:1]: is there a reason why your creating a new template?  couldn't you just re-use the [bastian.yaml template](https://github.com/Sage-Bionetworks/Agora-infra/blob/develop/templates/bastian.yaml)?

**Review by zaro0508 — CHANGES_REQUESTED** [review]

File: config/misc/agora-bastian-docdb.yaml (hunk 1 above)
Comment (emmanmills) [reply config/misc/agora-bastian-docdb.yaml:3]: Actually, these are your changes a while back. See [PR](https://github.com/Sage-Bionetworks/Agora-infra/commit/31a507e0ab8e1657e8924efbce8e111dfc23fee4). Should I just delete this file?

File: templates/bastian2.yaml (hunk 2 above)
Comment (emmanmills) [reply templates/bastian2.yaml:1]: You also created this file in this [PR](https://github.com/Sage-Bionetworks/Agora-infra/commit/31a507e0ab8e1657e8924efbce8e111dfc23fee4). Should I just delete it?

**Review by zaro0508 — APPROVED** [review]

**Pull request merged**